recipe/meta.yaml

```diff
@@ -22,10 +22,10 @@ requirements:
     - make
     - ninja
   host:
-    - mbedtls  # [not win]
+    - mbedtls =3.5
```

Comment:
Is this required to be exactly `=3.5` or is it sufficient to use `>=3.5`.

Reply:
Now that I understand `pin_compatible`,  `>=3.5` should work.